examples/ir_transmit.py

```diff
@@ -0,0 +1,23 @@
+"""IR transmit example using Circuit Playground Express"""
+import time
+import adafruit_irremote
+import pulseio
+import board
+import digitalio
+
+# Create a button object to trigger IR transmit
+button = digitalio.DigitalInOut(board.D4)
+button.direction = digitalio.Direction.INPUT
+button.pull = digitalio.Pull.DOWN
```

Comment:
should this be `digitalio.Pull.UP` to match essentials guide, etc? would require changing the conditional in the loop

Reply:
The buttons are backwards on the CPX. So either this or the loop will be backwards for CPX or an external button. We'll leave it as CPX for now and consider changing it if we receive a bunch of feedback.